index.js

```diff
@@ -256,7 +267,12 @@ module.exports = class Hypercore extends EventEmitter {
   }
 
   _addSession (s) {
+    if (this.sessions.lenth >= 1) {
```

Comment:
I think it's guaranteed to always contain at least 1 element, but not 100% sure that's true, so I added the if

Reply:
No longer relevant